tests/network/primary_pod_network.go

```diff
@@ -82,6 +86,59 @@ var _ = SIGDescribe("Primary Pod Network", func() {
 			It("should report PodIP as its own on interface status", func() { AssertReportedIP(vmi) })
 		})
 	})
+
+	Describe("Connectivity", func() {
+		const servicePort = 1500
+
+		AssertConnectivity := func(outboundVMI, inboundVMI *v1.VirtualMachineInstance) {
+			inboundVMIAddress := inboundVMI.Status.Interfaces[0].IP
+
+			By("Checking connectivity via ICMP")
+			Expect(tests.PingFromVMConsole(outboundVMI, inboundVMIAddress)).To(Succeed())
+
+			By("Checking connectivity via ICMP with MTU-sized frames")
+			mtu := getPodNetworkMTU(outboundVMI)
```

Comment:
Why do we need to read the MTU in a hacky way?
- Isn't it something expected here? (if not, why?)
- If not expected, then it there an API that can be queried?

Reply:
I would expect the mtu to be exposed as it is a nic property and the domxml allows setting it.
But my question is... why can't we assume that it is? Are there cases in the tests that this may not be the same?